config/default.yml

```diff
@@ -12,3 +12,8 @@ Style/DisableProgrammaticEnumValue:
   Description: 'Disallow users to create enums with an each_value statement'
   Enabled: pending
   VersionAdded: '0.93'
+
+Swagger/SpecValidation:
+  Description: 'Validates swagger spec for external endpoints'
+  Enabled: false
+  VersionAdded: '1.31.2' # Bump when actually released
```

Comment:
That is the doc that defines what cops are in this gem and what version they were released in.  It's used with a rake task called release where when you are happy to release a cop, you can run it and it will build out the appropriate updates to have it published to rubygems.

Reply:
sorry i meant is the versioning/release process for this documented somewhere in confluence?

Reply:
its actually documented right on this Github's README xD.  Theres a bit of extra info on the rubocop doc pages too.